src/components/StafferPage/StafferForm.jsx

```diff
@@ -157,22 +158,24 @@ const BaseCreateInstructorForm = ({
   );
 };
 
-BaseCreateInstructorForm.propTypes = {
+BaseStafferForm.propTypes = {
   handleSubmit: PropTypes.func.isRequired,
   pristine: PropTypes.bool.isRequired,
   submitting: PropTypes.bool.isRequired,
-  instructorOptions: PropTypes.shape({
+  stafferOptions: PropTypes.shape({
     data: PropTypes.shape(),
     error: PropTypes.string,
     isFetching: PropTypes.bool,
   }),
-  isCreating: PropTypes.bool,
+  isSaving: PropTypes.bool,
+  isCreateForm: PropTypes.bool,
 };
 
-BaseCreateInstructorForm.defaultProps = {
-  instructorOptions: {},
-  isCreating: false,
+BaseStafferForm.defaultProps = {
+  stafferOptions: {},
+  isSaving: false,
+  isCreateForm: false,
 };
 
-const CreateInstructorForm = reduxForm({ form: 'create-instructor-form' })(BaseCreateInstructorForm);
-export { basicValidate, extractOrgChoices, BaseCreateInstructorForm, CreateInstructorForm };
+const StafferForm = reduxForm({ form: 'create-staffer-form' })(BaseStafferForm);
+export { basicValidate, extractOrgChoices, BaseStafferForm, StafferForm };
```

Comment:
I think we should do export default on the connected form (`StafferForm`) and regular exports on the other stuff

Reply:
✅